src/app/templates/app/submit_new_license.html

```diff
@@ -104,6 +104,7 @@
       <label class="control-label col-sm-3" >&nbsp;OSI Status&nbsp;</label>
         <div class="col-sm-4">
           {{ form.osiApproved }}
+          <span class="help-block" id="OSIapproved" style="text-align: left; font-weight: bold; font-style: italic; cursor:pointer;" >Go to the <a href = "https://opensource.org/licenses/">OSI approved licenses</a> to know more. </span>
```

Comment:
Visit <a href = "https://opensource.org/licenses/">OSI approved licenses</a> to know more.